### ISSUE TITLE
Focus outline for Card

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-styles#2514]
Focus outline for Card - the space between the border and focus should be equal
<img width="981" alt="Screen Shot 2021-03-25 at 4 39 30 PM" src="https://user-images.githubusercontent.com/39598672/112540578-b9f76900-8d88-11eb-99a8-457a8ba02731.png">
